lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -2313,6 +2396,156 @@ def _containers(self) -> Dict[str, Container]:
         return {cont: self._charm.unit.get_container(cont) for cont in self._logs_scheme.keys()}
 
 
+class LogForwarder(ConsumerBase):
+    """Forward the standard outputs of all workloads operated by a charm to one or multiple Loki endpoints."""
+
+    def __init__(
```

Comment:
This should check if the Juju version is at least 3.4:
* something like [this](https://github.com/canonical/observability-libs/blob/main/lib/charms/observability_libs/v1/cert_handler.py#L413)
* compare JujuVersion objects explained [here](https://github.com/canonical/operator/blob/dc47640f8ea915049d9e20a1fe4ea189c96ae8d1/ops/jujuversion.py#L24)